script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_4_2.txt

```diff
@@ -1,23 +1,24 @@
 <Page1>
-...It's about my sister, Akiha.
+...Of course, I'm thinking about my sister Akiha.//　……やはり、妹の秋葉の事だ。choice from 1_4
 <Page2>
 In the beginning, when I was left at the Arima household, Akiha came to visit me several times.
-Unfortunately, she couldn't see me often because I had to go to the hospital every day, and after she went to a boarding school, we only exchanged a few letters.
+Unfortunately, she couldn't see me often because I had to go to the hospital every day. After she went off to a boarding school, we only exchanged a few letters.
 <Page3>
 Unlike Akiha, I'm no longer a member of the main family.//Since he was technically the eldest son of the Tohno family and was only booted out afterwards
 That was why I was able to lead a free and easy life.
 For the past seven years, I have had no contact with my sister.
+For these last seven years, I'm glad to say I've had no contact with my sister.//　ここ七年ばかり、妹との接点は皆無といってよかった。technically more like "my contact with my sister has been nil"
 <Page4>
-To be honest, I have no desire to return to the Tohno mansion.
-However, Akiha is there.
+To be honest, up to now, the idea that I want to return hadn't crossed my mind.//　正直なところ、いまさら遠野の家に戻りたいとは思ってはいなかった。
+But Akiha is there.
 As a child,
-Akiha was quiet, always timid as if she was holding something back, and followed me around anxiously.
+Akiha was quiet and always timid as if she were enduring something; she always depending on me and following me around.//　秋葉はおとなしくて、いつも何かを我慢しているように怯えていて、頼りなげに俺の後についてきた。
```

Comment:
**depended** on me and **followed** me around

Reply:
I still stand by "depending" and "following" but I forgot the word "was"